backendsrc/webDataHandler.js

```diff
@@ -1504,7 +1508,7 @@ class WebDataHandler {
             },
             {
                 "$group": {
-                    _id: "$closed_by.login",
+                    _id: "$closed_by.in",
```

Comment:
Is this supposed to be changed? It seems to break this function

Reply:
Nope, accidental change - good catch thanks! Will undo.